ui/components/app/flask/update-snap-permission-list/update-snap-permission-list.js

```diff
@@ -18,7 +19,7 @@ export default function UpdateSnapPermissionList({
     return getWeightedPermissions(t, permissions).map(
       ({ label, rightIcon, permissionName, permissionValue }) => (
         <div className={className} key={permissionName}>
-          <i className="fas fa-x" />
+          <Icon name={ICON_NAMES.CHECK} />
```

Comment:
I am not sure if we should be updating `fa-x` here? wdyt?

Reply:
I guess when I started editing the code this component was a check and that is why I updated it. I could revert to what it was.